Unstructured-IO/unstructured
# Feat: CORE-1269 - Install paddlepaddle wheel dependent on arch, supporting aarch64

Testing instructions

on Apple silicon

```
make docker-build
docker run -it unstructured:dev bash
python3
```
Then run the test in this PR https://unstructured-ai.atlassian.net/browse/CORE-1269

You should get output like shown in ticket

Run the same process on your local machine (not inside docker) with same test to verify the non aarch64 paddlepaddle got installed correctly
